tests/utils.go

```diff
@@ -103,6 +103,11 @@ const (
 	nfsPathCirros = "/nfsshare/cirros"
 )
 
+const (
+	RegistryDiskCirros = "kubevirt/cirros-registry-disk-demo:devel"
```

Comment:
These need to be replaced with calls to `tests.GetRegistryDisk`

Reply:
@rmohr rebased on top of your changes